cli/src/Api.ts

```diff
@@ -400,6 +413,18 @@ export default class Api {
     return this.entriesByIds(this._api.query.content.curatorGroupById)
   }
 
+  async allStakingLedgers(): Promise<[AccountId, Option<StakingLedger>][]> {
+    return await this.entriesByAccountIds(this._api.query.staking.ledger)
+  }
+
+  async isControllerValid(account: string) {
```

Comment:
```suggestion
  async isControllerValid(account: string): Promise<Option<StakingLedger>> {
```
Also, the name suggest it returns a boolean, so I suggest either renaming the function or adding `isSome` and changing the return type.

Reply:
:+1: